### PR TITLE
Revert "Created my Contact Page template"

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
Reverts Ntwali-Josue/UI-template#1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,10 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="css/style.css">
-    <title>Contact page </title>
+    <title>UI template</title>
```
Comment (ceelogre) [inline index.html:6]: UI template is not descriptive, find a better name.

Comment (TuyizeeAnastase) [inline index.html:6]: cany you @Ntwali-Josue  remove the second tittle?for UI template?

Comment (Ntwali-Josue) [reply index.html:6]: yeah i'm on it,i'll solve it as soon as possible

Comment (Ntwali-Josue) [reply index.html:6]: yeah i'm still doing on my landing and contact 
page it will be finished soon

**Pull request closed without merging**